diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index 48cf0d3..2d15bcc 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -330,4 +330,11 @@ function expand_(str, max, maxLength, maxDepth, depth, maxRewrites, isTop) {
     }
     return acc;
 }
-//# sourceMappingURL=index.js.map
\ No newline at end of file
+//# sourceMappingURL=index.js.map
+module.exports = expand;
+module.exports.expand = expand;
+module.exports.EXPANSION_MAX = exports.EXPANSION_MAX;
+module.exports.EXPANSION_MAX_LENGTH = exports.EXPANSION_MAX_LENGTH;
+module.exports.EXPANSION_MAX_DEPTH = exports.EXPANSION_MAX_DEPTH;
+module.exports.EXPANSION_MAX_REWRITES = exports.EXPANSION_MAX_REWRITES;
+module.exports.default = expand;
